### PR TITLE
Regional layer logic

### PR DESCRIPTION
**Pull request by vectorsize**
This makes it possible for selectors to select several layers upon selection, that way we can control additive vs exclusive selections.

### CODE REVIEW

**Comment by vectorsize** [comment]
closes #12

File: src/app/components/cesium/map/map.js [hunk 1]
```diff
@@ -56,6 +55,8 @@ class CesiumComponent extends Component {
 
     if (zoomLevel) this.handleZoom(zoomLevel)
     if (lockNavigation) return disablePanning(viewer)
+
+    this.state.viewer = viewer
```
Comment (sorodrigo) [inline src/app/components/cesium/map/map.js]: Why do you mutate the state?
Besides, I don't think it's needed cause after the call to mountMap you use the resulting viewer to setState inside bindMap.

File: src/app/components/explorable/explorable.js [hunk 2]
```diff
@@ -38,14 +34,15 @@ export const renderToggle = (layers, namespace = '') => toggle => (
 
 export const renderDropdown = sections => selectOption => (section, name) => {
   if (!sections[section]) return <span>{name}</span>
+  const { selections: sel } = sections[section]
   const { options, selected } = sections[section].selectors[name]
+  console.log(name, sections[section].selectors)
```
Comment (sorodrigo) [inline src/app/components/explorable/explorable.js]: no-console

File: src/app/pages/regional/regional-reducers.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import includes from 'lodash/includes'
 import difference from 'lodash/difference'
+import find from 'lodash/find'
```
Comment (sorodrigo) [inline src/app/pages/regional/regional-reducers.js]: why not use the native method?

**Pull request merged**